components/esb-tools/plugins/org.wso2.integrationstudio.gmf.esb.edit/src/org/wso2/integrationstudio/gmf/esb/provider/CallTemplateParameterItemProvider.java

```diff
@@ -190,6 +192,12 @@ public String getText(Object object) {
 
         if (parameterType.equalsIgnoreCase(RuleOptionType.VALUE.getName())) {
             if (parameterValue != null) {
+            	String operatingSystem = System.getProperty("os.name");
```

Comment:
Any possibility to move this "os.name" to a constant variable? If this system property is used in other places, shall we do so?

Reply:
Fixing in : [1123](https://github.com/wso2/integration-studio/pull/1123)